src/data/faq_de.json

```
@@ -1518,7 +1536,9 @@
                                     "<b>TAN-Hotline</b>",
                                     "Die TAN-Hotline hilft Ihnen bei der Eingabe der TAN, wenn Sie ein positives PCR-Testergebnis haben, und wenn Sie keine TAN oder kein Dokument mit PCR-Test-QR-Code erhalten haben. Sie erreichen sie von Montag bis Sonntag von 7 bis 20 Uhr Ortszeit in Deutschland:",
                                     "<b>National:</b> 0800 7540002",
-                                    "<b>International:</b> +49 30 498 75402"
+                                    "<b>International:</b> +49 30 498 75402",
+                                    "Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ?????h eingestellt.",

```

Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ??????h eingestellt.",
```